tower.yml

```
@@ -0,0 +1,5 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
+  samplesheet.csv:
+    display: "Auto-created samplesheet with collated metadata and FASTQ paths"
```

I haven't tested this on Tower but my understanding from https://help.tower.nf/22.1/reports/overview/#providing-reports is that this `tower.yml` should have the full (relative) paths
```suggestion
  multiqc/multiqc_report.html:
    display: "MultiQC HTML report"
  genomenote_info/samplesheet_valid.csv:
    display: "Auto-created samplesheet with collated metadata and FASTQ paths"
```